### DISCUSSION TITLE
Latest version added a floating bar / how to disable it?

### DISCUSSION THREAD

**Discussion by quasiblob** [heshengtao/comfyui_LLM_party#118] [General]
Hi.

Seems like updating to the latest version added this bar. Is it somehow possible to disable this floating bar? I don't have use for it, I would rather not see it as it is very distracting.

![image](https://github.com/user-attachments/assets/87874aaa-b187-4bb7-9de5-1ac390fd18ef)

**Comment by heshengtao** [comment]
![1729249369812](https://github.com/user-attachments/assets/8a1a96cf-6cbc-4732-9c6c-35442da231f9)
I just added a put away button. This thing can be dragged and dropped, you can put it anywhere, and it is still black, so it can be thrown into your comfyui management bar. Like this, please find the drag bar in the picture hahaha.

**Reply by quasiblob** [reply]
I just updated the custom nodes, saw that there was an update to LMM party. I tried dragging it to new UI tool bar, but with a few tries I didn't manage to do that.  Is there some special way I need to do that?

**Reply by quasiblob** [reply]
To make it clear - I can't get it to dock like in your picture, however Comfy UI queue buttons can be docked.

**Comment by heshengtao** [comment]
The reason this slider is called a slider is that it doesn't dock. Because I want to put many buttons inside it in the future, I don’t want to write them into this sidebar, which is too small. Just because you don’t use it doesn’t mean others won’t. I’m not putting ads on the frontend; I’ve hidden them away, and it might be hard for others to find them. Why are you so selfish? Does the world revolve around you?

All these features are great for intelligent agents—one for managing API keys, another for quickly turning workflows into APIs, and one for rapidly deploying workflows into conversational applications for debugging agents. My project homepage clearly states that this plugin is for creating intelligent agents. Shouldn’t I make the features for publishing agent applications and APIs the most prominent?

You’re giving orders here, but what contributions have you made to the open-source community?
